beacon/types/config.go

```diff
@@ -122,16 +134,26 @@ func (c *ChainConfig) ForkAtEpoch(epoch uint64) Fork {
 
 // AddFork adds a new item to the list of forks.
 func (c *ChainConfig) AddFork(name string, epoch uint64, version []byte) *ChainConfig {
+	knownIndex := math.MaxInt
+	for i, fname := range knownForks {
+		if name == fname {
+			knownIndex = i
+			break
+		}
+	}
```

Comment:
You can use `slices.Index(knownForks, name)` here.

Reply:
I learned something today :)